schemas/documents/seo.js

```diff
@@ -0,0 +1,38 @@
+import { MdSearch } from "react-icons/md";
+
+export default {
+  title: "SEO Settings",
+  name: "seoSettings",
+  icon: MdSearch,
+  type: "object",
+  fields: [
+    {
+      title: "SEO Title",
+      name: "seoTitle",
+      description: "This will override the page title",
+      type: "string"
+    },
+    {
+      title: "Keywords",
+      name: "keywords",
```

Comment:
can we make it all uniform by prefixing names with `seo`, so in this case, it should be `seoKeywords`. Use `camelCase` as well.

Reply:
changed `keywords` to `seoKeywords`